include/mbedtls/pk_info.h

```diff
@@ -0,0 +1,274 @@
+/**
+ * \file pk_info.h
+ *
+ * \brief Public Key cryptography abstraction layer: object interface
```

Comment:
I think this should be “engine interface”, not “object interface”. “Object interface” makes it seem like this is alternative way to do the same thing as some other interface (a procedural interface, presumably). While technically true, we don't want applications to call this interface, so we should make it clear that this file isn't for them.

Reply:
Done